### PR TITLE
Add Delta Time Variable for Episodes

### PR DESCRIPTION
**Pull request by BKSteve**
Attempt to Avoid #7390 

Proposed changes in this pull request:
Add new variable in days to `Search settings` which is used to set episodes older than current date minus the set days variable. 

Totally untested. It was in the wrong place initially but fingers crossed should work.

Add:
This is only half tested (mako tested) the actual loading and adding new shows still to test.

### CODE REVIEW

File: sickchill/oldbeard/dailysearcher.py [hunk 1]
```diff
@@ -70,14 +71,15 @@ def run(self, force=False):
             with ep.lock:
                 prefix = _("New episode {episode_string} airs today,").format(episode_string=ep.pretty_name)
                 if ep.season == 0:
-                    logger.info(_("{prefix} setting status to SKIPPED because is a special season").format(prefix=prefix))
+                    logger.info(
```
Comment (miigotu) [inline sickchill/oldbeard/dailysearcher.py]: Was this missing the indent?

Comment (BKSteve) [reply sickchill/oldbeard/dailysearcher.py]: Pycharm did it's thing on long lines. Thought I'd caught them all.

File: sickchill/tv.py [hunk 2]
```diff
@@ -1639,9 +1639,12 @@ def loadFromIndexer(self, season=None, episode=None):
...
                 self.status = UNAIRED
             elif self.status in [UNAIRED, UNKNOWN]:
+                # Do some date calc
+                delta_time = datetime.date.today() - datetime.timedelta(days=settings.SHOW_SKIP_OLDER)
                 # Only do UNAIRED/UNKNOWN, it could already be snatched/ignored/skipped, or downloaded/archived to disconnected media
-                logger.debug(f"Episode has already aired, marking it {statusStrings[self.show.default_ep_status]}")
-                self.status = self.show.default_ep_status if self.season > 0 else SKIPPED  # auto-skip specials
+                # auto-skip specials and check date in delta period too.
+                self.status = self.show.default_ep_status if self.season > 0 and self.airdate > delta_time else SKIPPED
+                logger.debug(f"Episode has already aired, marking it {statusStrings[self.status]}")
```
Comment (miigotu) [inline sickchill/tv.py]: I'd do the delta_time separate, so you can use a different logger line that more accurately says what it's doing.

Comment (BKSteve) [reply sickchill/tv.py]: Noted. Will help when doing new show adds per discord question. 
https://discord.com/channels/@me/826748317902962688/1123209837791158413

Comment (BKSteve) [reply sickchill/tv.py]: Broke the logic out and specific messages

**Comment by miigotu** [comment]
You need to `poe format` to fix formatting test errors.

**Review by miigotu — APPROVED** [review]

**Pull request merged**